src/accelerator.rs

```diff
@@ -0,0 +1,87 @@
+use drivers::Driver;
+use motions::Motion;
+
+fn clamp(n: isize, min: isize, max: isize) -> isize {
+  if n < min {
+    return min;
+  }
+  if n > max {
+    return max;
+  }
+  n
```

Comment:
I noticed, it doesn't have a semicolon 😊
On Sun, Dec 20, 2015 at 11:08 AM Victor M. Suarez <notifications@github.com>
wrote:

> In src/accelerator.rs
> <https://github.com/calebmer/accelerate/pull/2#discussion_r48104873>:
>
> > @@ -0,0 +1,87 @@
> > +use drivers::Driver;
> > +use motions::Motion;
> > +
> > +fn clamp(n: isize, min: isize, max: isize) -> isize {
> > +  if n < min {
> > +    return min;
> > +  }
> > +  if n > max {
> > +    return max;
> > +  }
> > +  n
>
> n is the input, in rust you dont have to return n; you can just leave it
> at n and it means the same
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/calebmer/accelerate/pull/2/files#r48104873>.
>